src/lightning/pytorch/callbacks/gradient_accumulation_scheduler.py

```diff
@@ -121,10 +121,9 @@ def on_train_start(self, trainer: "pl.Trainer", pl_module: "pl.LightningModule")
             )
 
         # local import to avoid circular import
-        from lightning.pytorch.accelerators import IPUAccelerator
         from lightning.pytorch.strategies import DeepSpeedStrategy
 
-        unsupported_accelerators = (IPUAccelerator,)
+        unsupported_accelerators = set()
```

Comment:
The list here is empty, so you can completely remove this variable and the corresponding if-statement below